Wikipedia/Code/WMFSettingsTableViewCell.m

```diff
@@ -45,6 +51,17 @@ - (void)setIconName:(NSString *)iconName {
     }
 }
 
+- (void)setIconOriginalRendering:(BOOL)iconOriginalRendering {
+    _iconOriginalRendering = iconOriginalRendering;
+    if (iconOriginalRendering) {
+        self.titleIcon.image = [[UIImage imageNamed:self.iconName] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
+        self.titleIcon.contentMode = UIViewContentModeScaleAspectFit;
+    } else {
+        self.titleIcon.image = [[UIImage imageNamed:self.iconName] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+        self.titleIcon.contentMode = UIViewContentModeScaleAspectFill;
+    }
+}
+
 - (void)setDisclosureText:(NSString *)disclosureText {
```

Comment:
I think these changes are causing the YiR and Article Storage icons to resize

<img width="300" height="700" alt="Image" src="https://github.com/user-attachments/assets/05333031-fd9a-44e6-b31d-ab6566b3584f" />